reference/cd-azure-devops.md

```diff
@@ -0,0 +1,277 @@
+---
+title: Azure DevOps
+---
+
+This page details how to use [Azure DevOps](https://azure.microsoft.com/en-us/services/devops/) to manage deploying
+stacks based on commits to specific Git branches, and based on the build reason. You may also choose to introduce a
+[Manual Intervention](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/manual-intervention?view=vsts) task to control the preview vs. update step for any pulumi stack.
+
+Pulumi doesn't require any particular arrangement of stacks or workflow to work in a
+continuous integration / continuous deployment system. So the steps described here can be
+altered to fit into any existing type of deployment setup.
+
+This page shows you how to use the YAML method to configure your DevOps pipeline, but you can easily adapt
+the steps outlined in the sample YAML file below to the Visual Designer as well.
+
+## Prerequisites
+- An account on https://app.pulumi.com.
+- The latest CLI.
+  - Installation instructions are [here](https://pulumi.io/quickstart/install.html).
+- A bare repo and set the remote URL to be your Azure DevOps project.
```

Comment:
The "... and set the remote ..." part sounds like it isn't actually a prerequisite. Do you think it would be clearer to say "A Git repository with your DevOps project set as the remote" instead?